src/pages/dashboard/_UsPositiveAndTotalTestsContainer.js

```diff
@@ -28,6 +15,18 @@ export default function UsAreaChartContainer() {
       }
     }
   `)
+  const transformedData = useMemo(() => {
```

Comment:
technically `data` will NEVER change so using `useMemo` here is unnecessary.
The data comes in at build time once and is hard coded in the built file.